project-base/storefront/tailwind.config.js

```diff
@@ -59,6 +59,9 @@ module.exports = {
             fontSize: {
                 clamp: 'clamp(16px, 4vw, 22px)',
             },
+            borderColor: {
+                transparent: 'rgba(0, 0, 0, 0.4)',
+            },
```

Comment:
What is the purpose of this? I am not a tailwind specialist, so I am not able to tell if this is correct or not, but could you please elaborate?

Reply:
This was an initial hack to have a transparent border, but the name was chosen poorly and found a way how to achieve this natively, removed.